std_msgs/Header header
float32 signal_left
float32 signal_middle
float32 signal_right
